.circleci/config.yml.old

```diff
@@ -0,0 +1,73 @@
+# Python CircleCI 2.0 configuration file
```

Comment:
do we need to keep the old configuration stored in git? its contained in the history of the repo

Reply:
nope